### PR TITLE
test.py: remove print(...)

### PR DESCRIPTION
**Pull request by gureks**
Removed the unnecessary `print()` from
`test_caching_multi_results/test.py`

Fixes https://github.com/coala/coala/issues/4071

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [X] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [X] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: tests/misc/test_caching_multi_results/test.py [hunk 1]
```diff
@@ -1,2 +1 @@
 a = 3
-print(a)
```
Comment (adhikasp) [inline tests/misc/test_caching_multi_results/test.py]: I think you shouldn't delete the line entirely, just leave `a`.

The reason is the test that using this file (`CachingTest.py::test_caching_multi_results()`) expect this file has more than 1 lines and thrown an error (IDK why the CI not failing, maybe the test itself is faulty?)

Comment (gureks) [inline tests/misc/test_caching_multi_results/test.py]: The build now failed case `a` was not defined. Previously it was correct.

**Comment by sils** [comment]
ack https://github.com/coala/coala/pull/4480/commits/507aba29fc8758385e9ae5b31128207f08bd637b

**Comment by sils** [comment]
ack f60b1f8 rebased for you to merge this quick

**Comment by sils** [comment]
@rultor merge

**Pull request merged**